tests/rspec/spec/meta-tests/config_file_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'config_file'
+
+describe ConfigFile do
+  before(:all) do
+    @config_file = ConfigFile.new(File.join(ENV['RSPEC_PATH'], '../smoke/aws/vars/aws.basic.yaml'))
+  end
+
+  it 'returns the right node count' do
+    expect(@config_file.node_count).to eq(5)
```

Comment:
Do you think it's a little brittle having hardcoding this value separately from the fixture, no?

Reply:
Sorry, what do you mean by "fixture"?

Reply:
Hardcoding the "5" is definitely not perfect. I am happy for alternative suggestions.

Reply:
The separate file containing the test content. Rather than hardcoding the 5, it may be better to construct a test case in code, as we often do in go. That way the expected value is coupled to the test case. Unfortunately, the ConfigFile class can only be instantiated from a file :/